### PR TITLE
	new file:   packages/speechd/build.sh

### PR DESCRIPTION
**Pull request by SDRausty**
# new file:   packages/speechd/build.sh

ref https://github.com/termux/termux-packages/issues/6606

### CODE REVIEW

**Comment by SDRausty** [comment]
**Link to home page and sources**

1. Home page: https://freebsoft.org/speechd
2. Source code: https://github.com/brailcom/speechd

**Additional information**
Have you compiled or tried to compile the package on device?

Yes, in [TermuxArch](https://termuxarch.github.io/TermuxArch/) with the command `yay speech-dispatcher --noconfirm` which builds the Arch Linux Speech Dispatcher package on device in Termux PRoot.

Speech Dispatcher also installs as an Arch Linux package without the need to compile and appears to function with [TDSR (A console screen reader for macOS and Linux)](https://github.com/tspivey/tdsr) on device in Termux PRoot:
```
[user18:24tdsr]$ ./tdsr
[user@localhost tdsr]$
```

File: packages/speechd/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/brailcom/speechd
+TERMUX_PKG_DESCRIPTION="Common high-level interface to speech synthesis"
+TERMUX_PKG_LICENSE="GPL2, GPL3 and LGPL"
```
Comment (xeffyr) [inline packages/speechd/build.sh:3]: Use comma-separated (no "and") SPDX license identifiers here.

See https://github.com/termux/termux-packages/blob/master/scripts/lint-packages.sh#L18-L36.

**Review by xeffyr — CHANGES_REQUESTED** [review]

**Comment by SDRausty** [comment]
modified:   packages/speechd/build.sh

File: packages/speechd/build.sh (hunk 1 above)
Comment (SDRausty) [reply packages/speechd/build.sh]: Thank you for the suggestion; modified as requested.

**Comment by Grimler91** [comment]
It needs some patches and depends on `dotconf`. I pushed what I have to a new branch: https://github.com/termux/termux-packages/pull/6624

**Review by oakleela — APPROVED** [review]

**Comment by SDRausty** [comment]
### [This pull request is still a work in progress.](https://github.com/termux/termux-packages/pull/6624)

[The proposed draft for `speechd` is at "Add speechd and dependency dotconf #6624."](https://github.com/termux/termux-packages/pull/6624)

**Pull request closed without merging**